### ISSUE TITLE
Should renderMode=light replace the element's child

### ISSUE THREAD

**Issue by matthewp** [matthewp/bram#40]
I think the answer is yes, however, if you had a template like:

```html
<template>
  Hello {{name}}
</template>
```

And an element like:

```js
class MyElement extends Bram.Element {
  static get template() {
    return document.querySelector('template');
  }
  static get renderMode() {
    return 'light';
  }
}
customElements.define('my-element', MyElement);
```

Being used like:

```html
<my-element>Some inner stuff</my-element>
```

Would you expect it to replace the inner contents with the template or just append? Currently it appends, but that feels wrong to me.

**Comment by bedeoverend** [comment]
My two bobs - I agree, should replace. Appending feels a little magical, and hard to keep track of the DOM state. If an element renders into it's light DOM, it makes sense that the element itself would dictate all of it's light DOM - the author should just document as such. Also seems a bit arbitrary if it appends the template, as opposed to prepending, or injecting somewhere else in the content...though that might just be me being overly pedantic.

**Comment by matthewp** [comment]
Cool, I agree. Yeah, there's no reason to keep the children in place.  I'll go ahead and fix this one.

**Comment by matthewp** [comment]
Released as 0.6.2
